python/ray/train/_internal/worker_group.py

```diff
@@ -87,14 +87,14 @@ def construct_metadata() -> WorkerMetadata:
     node_id = ray.get_runtime_context().get_node_id()
     node_ip = ray.util.get_node_ip_address()
     hostname = socket.gethostname()
-    resource_ids = ray.get_runtime_context().get_resource_ids()
+    accelerator_ids = ray.get_runtime_context().get_accelerator_ids()
```

Comment:
@matthewdeng I deprecated get_resource_ids() and introduced get_accelerator_ids() as we discussed. You may want to change train side as well.